### ISSUE TITLE
cache: warnings if libelektra-cache.so not installed

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#2695]
```sh
cd /usr/lib/x86_64-linux-gnu/elektra4 && sudo mv libelektra-cache.so libelektra-cache.so-backup
```

(or do not install libelektra-cache.so), then do something with KDB:

```sh
 Sorry, 5 warnings were issued ;(
 Warning (#1):
        Description: could not load module, dlopen failed
        Module: dl
        At: /home/jenkins/workspace/libelektra_master-Q2SIBK3KE2NBEMJ4WVGJXAXCSCB77DUBUULVLZDKHQEV3WNDXBMA@2/libelektra/src/libs/loader/dl.c:89
        Reason: of module: libelektra-cache.so, because: libelektra-cache.so: cannot open shared object file: No such file or directory
        Mountpoint: system/elektra
        Configfile: kdbOpen(): mountGlobals
 Warning (#64):
        Description: could not load plugin in process plugin
        Module: kdb
        At: /home/jenkins/workspace/libelektra_master-Q2SIBK3KE2NBEMJ4WVGJXAXCSCB77DUBUULVLZDKHQEV3WNDXBMA@2/libelektra/src/libs/elektra/mount.c:314
        Reason: cache
        Mountpoint: system/elektra
        Configfile: kdbOpen(): mountGlobals
 Warning (#1):
        Description: could not load module, dlopen failed
        Module: dl
        At: /home/jenkins/workspace/libelektra_master-Q2SIBK3KE2NBEMJ4WVGJXAXCSCB77DUBUULVLZDKHQEV3WNDXBMA@2/libelektra/src/libs/loader/dl.c:89
        Reason: of module: libelektra-cache.so, because: libelektra-cache.so: cannot open shared object file: No such file or directory
        Mountpoint: system/elektra
        Configfile: kdbOpen(): mountGlobals
 Warning (#64):
        Description: could not load plugin in process plugin
        Module: kdb
        At: /home/jenkins/workspace/libelektra_master-Q2SIBK3KE2NBEMJ4WVGJXAXCSCB77DUBUULVLZDKHQEV3WNDXBMA@2/libelektra/src/libs/elektra/mount.c:314
        Reason: cache
        Mountpoint: system/elektra
        Configfile: kdbOpen(): mountGlobals
 Warning (#139):
        Description: failed to mount global plugins
        Module: 
        At: /home/jenkins/workspace/libelektra_master-Q2SIBK3KE2NBEMJ4WVGJXAXCSCB77DUBUULVLZDKHQEV3WNDXBMA@2/libelektra/src/libs/elektra/kdb.c:312
        Reason: Mounting global plugins failed
        Mountpoint: system/elektra
        Configfile: kdbOpen(): mountGlobals
```

**Comment by markus2330** [comment]
Expected behavior: No warning but some extra tool tells that cache does not work.

**Comment by markus2330** [comment]
The easiest fix is probably that we define "cache" to be in the minimal set of plugins? (Then a warning would be okay.)

**Comment by mpranj** [comment]
The original issue has actually been resolved since 0.9.0 (64932464598fff8bfb107004deeb5ec0ec2bde40) and I can not reproduce it any more.
